packages/react-components/react-tags/library/src/components/InteractionTagPrimary/useInteractionTagPrimaryStyles.styles.ts

```diff
@@ -279,7 +297,7 @@ export const useInteractionTagPrimaryStyles_unstable = (
 
   const rootCircularContrastStyles = useRootCircularContrastStyles();
 
-  const { shape, size, appearance } = state;
+  const { shape, size, appearance, selected } = state;
```

Comment:
_nit: Should we just add `disabled` here so we don't have to keep using `state.disabled` below, and can use just `disabled`?

Reply:
fixed